content/_index.fr.md

```diff
@@ -8,10 +8,10 @@ hero:
       author: Observatoire de l'innovation dans le service public de l'OCDE
       title: Transformer la loi en code « améliore l'impact des politiques publiques et la fourniture de service »
       link: https://oecd-opsi.org/publications/cracking-the-code/
-    - image: dpg.png
-      author: Digital Public Goods Alliance
-      title: Bien public numérique candidat
-      link:
+    - image: wgs.png
+      author: World Governement Summit
```

Comment:
Please use the suggestions to avoid repeated feedback 😉 
```suggestion
      author: World Government Summit
```

Reply:
Really sorry about that, I would love a sync point to see with you how to best use the GitHub suggestions.